utils.go

```diff
@@ -11,3 +11,12 @@ func AmountToString(amount *Amount) string {
 func AmountToDecimalString(amount *Amount) string {
 	return fmt.Sprintf("%.2f", float64(amount.Amount)/100.0)
 }
+
+// TruncateString returns a prefix of str of length truncateLength or all of
+// str if truncateLength is greater than len(str)
+func TruncateString(str string, truncateLength int) string {
```

Comment:
For our go-report card later can we also unit test this?

Reply:
Added some tests